addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <div class="o_AttachmentLinkPreview" t-att-class="{'o-isCompact': props.isCompact}">
+            <t t-if="attachmentLinkPreview and attachmentLinkPreview.attachment">
+                <t t-if="attachmentLinkPreview.imageUrl">
```

Comment:
This currently doesn't work, it generates the imageUrl even if there is no image.